### PR TITLE
Add summary

### PR DESCRIPTION
**Pull request by johnwatson484**
https://eaflood.atlassian.net/browse/SFI-2232

### CODE REVIEW

File: app/generator/conversion.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const CONVERSION_RATE = 2.835
+
+const millimetresToPoints = (millimetres) => {
+  return millimetres * CONVERSION_RATE
+}
```
Comment (johnwatson484) [inline app/generator/conversion.js:5]: GOV.UK Notify has strict requirements for margins and placement of the address.  All of those measurements are in millimetres.  `pdfmake` works in points, so have added this to convert mm to points.

To validate I have also measured the placement of address and margins to ensure they are "safe" using Adobe acrobat's measuring tools.

Letter specs -> https://www.notifications.service.gov.uk/using-notify/guidance/letter-specification

File: app/generator/get-definition.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 const getDefinition = (statement) => {
+  const topMargin = millimetresToPoints(5)
+  const sideMargin = millimetresToPoints(15)
   return {
     pageSize: A4,
     content: generateContent(statement),
     styles,
-    defaultStyle: styles.default
+    defaultStyle: styles.default,
+    pageMargins: [sideMargin, topMargin, sideMargin, topMargin]
```
Comment (johnwatson484) [inline app/generator/get-definition.js:14]: Ensuring fits within notify boundaries for letters

File: app/generator/content/summary/get-sbi.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    columns: [
+      { width: 200, text: 'Single business identifier (SBI):' },
+      { width: '*', text: sbi }
+    ],
+    style: 'column',
+    columnGap: 10
+  }
+}
+
+module.exports = getSBI
```
Comment (AbidemiAdio) [inline app/generator/content/summary/get-sbi.js:12]: It looks like `sbi` and `businessName` column will always have the same formatting style, we may consider creating column templates for similar columns.

File: app/generator/content/summary/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      { image: `${imagePath}/rpa-logo.jfif`, fit: [millimetresToPoints(200), millimetresToPoints(25)], style: 'logo' },
+      getAddress(statement.businessName, statement.address),
+      { text: `${statement.scheme.name} ${statement.scheme.year}`, style: 'header1' },
+      { text: 'Payment statement', style: 'subTitle' },
+      getBusinessName(statement.businessName),
+      getSBI(statement.sbi),
+      `\n\nThis statement explains your payment for the ${statement.scheme.name} (${statement.scheme.shortName}). It is made up of 3 parts.`,
+      '\nPart 1 provides a summary of the most recent payment.',
+      'Part 2 explains how we calculated the payment.',
+      'Part 3 highlights were to go for more information.'
```
Comment (AbidemiAdio) [inline app/generator/content/summary/index.js:20]: We may consider separating large static text/content from code file for better readability.

**Review by AbidemiAdio — DISMISSED** [review]
QED

**Pull request merged**